src/core/utils/EventNormalizer.ts

```diff
@@ -1,3 +1,5 @@
+import { Signal } from '../types/Flux';
```

Comment:
I'm sorry but I really dislike the term ```Flux```. At Odoo we have always used ```Flow``` instead. Not saying you are wrong, just that I'm not used to it. That being said, I wouldn't really like calling the file ```Flow``` either. Personally, I would rename it ```Actions``` and make it a `d.ts` file so that it is automatically imported everywhere in the project.